pc/util.js

```diff
@@ -0,0 +1,26 @@
+var fs = require('fs');
+
+exports.readTextFileToArray = function (path) {
+    if (fs.existsSync(path)) {
+        var buf = fs.readFileSync(path);
+        return buf.toString().split("\n");
+    }
+    console.error('File not found');
+    return [];
+}
+
+exports.extractTestCase = function (testCase) {
```

Comment:
Because this function is only used in string-stripper.spec.ts I think it would make more sense to have it there instead of in util

Reply:
Oh yes, I agree with that.